lib/cisco_node_utils/cmd_ref/fabricpath.yaml

```diff
@@ -0,0 +1,178 @@
+# fabricpath
+---
+# Fabricpath feature is not available on N3K and N9K
+_exclude: [/N3/, /N9K/]
```

Comment:
I purposely excluded the K, to allow for more model variations just like N7700 and N7K (there are 2 variations for N7K)

Thx,
deepak

From: Mike Wiebe <notifications@github.com<mailto:notifications@github.com>>
Reply-To: cisco/cisco-network-node-utils <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Friday, November 13, 2015 at 7:02 AM
To: cisco/cisco-network-node-utils <cisco-network-node-utils@noreply.github.com<mailto:cisco-network-node-utils@noreply.github.com>>
Cc: Cisco Employee <dcherian@cisco.com<mailto:dcherian@cisco.com>>
Subject: Re: [cisco-network-node-utils] adding fabricpath topo and fixed feature_set as a class method (#75)


In lib/cisco_node_utils/cmd_ref/fabricpath.yaml<https://github.com/cisco/cisco-network-node-utils/pull/75#discussion_r44790542>:

> @@ -0,0 +1,178 @@
> +# fabricpath
> +---
> +# Fabricpath feature is not available on N3K and N9K
> +_exclude: [/N3/, /N9K/]


Are you missing the 'K' for N3K?

-
Reply to this email directly or view it on GitHub<https://github.com/cisco/cisco-network-node-utils/pull/75/files#r44790542>.